<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отсечение отрезков выпуклым отсекателем</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_cut">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="paint">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>560</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсечь</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_lock">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>530</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_polygon">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввести отрезки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cutter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввести отсекатель</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_polygon">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>НАЧАЛО </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>КОНЕЦ</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="line_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>470</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить параллельный отрезок</string>
    </property>
   </widget>
   <widget class="QLabel" name="coord">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
